### PR TITLE
Better download page & promo

### PR DESCRIPTION
**Pull request by kkuepper**

### CODE REVIEW

**Review by sifferhans — APPROVED** [review]
Looks good!

File: pages/download.vue [hunk 1]
```diff
@@ -10,16 +10,28 @@ const links = await api.downloadLinksGet();
 
 function isAppleSilicon() {
   try {
-    const canvas = document.createElement("canvas");
-    const gl = canvas.getContext("webgl");
+    // Best guess if the device uses Apple Silicon: https://stackoverflow.com/a/65412357
```
Comment (SimonSimCity) [inline pages/download.vue:13]: I'd rather like to use an extension here. The benefit is that other people will quickly get edge cases and improve it for us. The here linked stackoverflow question has an answer with a link to this package: https://github.com/faisalman/ua-parser-js

**Review by SimonSimCity — APPROVED** [review]
Just that one suggestion. Since I'm not into differentiating OS'ses in JavaScript, I'm fine with either option.

File: pages/download.vue (hunk 1 above)
Comment (kkuepper) [reply pages/download.vue:13]: I agree with you in principle and we're actually using ua-parser-js for differentiating the OS. But for detecting Apple Silicon ua-parser-js relies on a new browser feature that only works on Chrome. I created an issue with them: https://github.com/faisalman/ua-parser-js/issues/732

File: components/sidebar/DesktopPromo.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+const profileStore = useProfileStore();
+</script>
+
+<template>
+  <div v-if="!profileStore.hasDisabledDownloadPromo" class="h-[160px]">
+    <div class="promo-shadow absolute bottom-3 left-3 right-3 text-black-1">
+      <div class="flex flex-col gap-2 border-b-2 border-label-separator p-3">
+        <div class="flex justify-between">
+          <NuxtIcon name="icon.download" class="text-2xl" />
+          <div></div>
```
[the remark this answers is not in the document]
Comment (kkuepper) [reply components/sidebar/DesktopPromo.vue:11]: Removed

**Pull request merged**